compiler/int128.nim

```diff
@@ -345,11 +345,12 @@ proc low64(a: Int128): uint64 =
   bitconcat(a.udata[1], a.udata[0])
 
 proc `*`*(lhs, rhs: Int128): Int128 =
+  {.push warnings: off.}  # Warning: target type is larger than source type.
```

Comment:
instead of disabling warnings, using `uint64(lhs.udata[1])` should produce the desired effect without a warning

Reply:
No, your suggested code does not compile. :shrug:

Reply:
oof, sorry, that's an array .. weird code - anyway: `uint64(cast[uint32](lhs.udata[1]))`

Reply:
```nim
  let a32 = uint64(cast[uint32](lhs.udata[1])) 
  let a00 = uint64(cast[uint32](lhs.udata[0]))
  let b32 = uint64(cast[uint32](lhs.udata[1])) 
  let b00 = uint64(cast[uint32](lhs.udata[0])) 
```

Like that?, does not compile, compiler wont compile.

Reply:
```
  let a32 = uint64(lhs.udata[1])
  let a00 = uint64(lhs.udata[0])
  let b32 = uint64(rhs.udata[1])
  let b00 = uint64(rhs.udata[0])
```
so going back to my original suggestion - this compiles for me on 1.6 - what error are you getting?